htdocs/web_portal/javascript/linking.js

```diff
@@ -85,7 +85,7 @@ function getRegExIdString() {
     // End with @iris.iam.ac.uk
     } else if (authType === "IRIS IAM - OIDC") {
         // var regExIdString = /^([a-f0-9]{8}\-[a-f0-9]{4}\-[a-f0-9]{4}\-[a-f0-9]{4}\-[a-f0-9]{12})@iris\-iam\.ac\.uk$/;
-        var regExIdString = /^.+@iris\-iam\.ac\.uk$/;
+        var regExIdString = /^.+@iris\-iam\.stfc\.ac\.uk\/$/
```

Comment:
My regex is rusty but did you intend to force a trailing slash? Added line termination semi-colon.
```suggestion
        var regExIdString = /^.+@iris\-iam\.stfc\.ac\.uk$/;
```

Reply:
I do mean to force a trailing slash, it's included in the ID string the IRIS IAM gives out. Good spot about the `;` :)

Reply:
For a future change maybe but this very much feels like it shouldn't be hardcoded.